spec/asynciterable-operators/toobservable-spec.ts

```diff
@@ -1,11 +1,11 @@
 import * as Ix from '../Ix';
+const { symbolObservable } = (Ix as any).default;
```

Comment:
Why does this have to be imported like this with a cast to `any`?

Reply:
I mean Typescript. We use Terser for the es2015+ umd bundles, but closure for the es5 umd. Even with all its warts it's still the best option at the moment, because its compiled iterators are as fast (or sometimes faster, in the case of FF/Edge) as native iterators in es2015+ envs.